flask_appbuilder/security/sqla/manager.py

```diff
@@ -84,6 +92,13 @@ def get_session(self):
         return self.appbuilder.get_session
 
     def register_views(self):
+        if self.appbuilder.app.config.get("ENABLE_USER_CRUD_API", False):
```

Comment:
now as we have renamed these apis as security apis, I wonder if we should just reuse existing `FAB_ADD_SECURITY_VIEWS` for this too.

Reply:
agree, and it also makes it consistent with FAB feature flags.